### PR TITLE
added NO_CLEAN_UP option for E2E tests

### PR DESCRIPTION
**Pull request by BPylypenko**
E2E tests teardown is not executed when the `NO_CLEAN_UP` variable has another value than `False/false`.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -237,6 +238,7 @@ cleanup-e2e-robot:
 ## e2e-robot: Run e2e robot tests
 e2e-robot:
 	pabot --verbose --processes ${ROBOT_THREADS} \
+		  -v NO_CLEAN_UP:${NO_CLEAN_UP} \
```
Comment (lev-titov) [inline Makefile:241]: indent

**Review by lev-titov — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (BPylypenko) [reply Makefile:241]: On my PC, it's showing as should be:
image

File: packages/tests/e2e/robot/resources/keywords.robot [hunk 2]
```diff
@@ -138,6 +138,19 @@ Cleanup resource
...
+E2E test setup
+    [Arguments]  ${example_id}
+    Set Environment Variable  ODAHUFLOW_CONFIG  ${LOCAL_CONFIG}
+    Login to the api and edge
+    Cleanup example resources  ${example_id}
+
+E2E test teardown
+    [Arguments]  ${example_id}
+    ${NO_CLEAN_UP_lower}  Evaluate  '${NO_CLEAN_UP}'.lower()
+    Pass Execution If  '${NO_CLEAN_UP_lower}' != 'false'  Suite Teardown is not executed
```
Comment (keshamin) [inline packages/tests/e2e/robot/resources/keywords.robot]: Is there a way to operate with `NO_CLEAN_UP` parameter as `bool`, not `string`? If yes, please implement it that way.

**Review by keshamin — APPROVED** [review]

File: packages/tests/e2e/robot/resources/keywords.robot (hunk 2 above)
Comment (BPylypenko) [reply packages/tests/e2e/robot/resources/keywords.robot]: OK, I'll try. If it's not possible I will add the comment in `keywords.robot` file.

**Pull request merged**